src/include/ndpi_typedefs.h

```diff
@@ -1373,6 +1372,10 @@ struct ndpi_flow_struct {
     u_int16_t num_processed_pkts;
   } stun;
 
+  struct {
+    u_int8_t certificate_processed:1, _pad:7;
+  } tls_quic; /* Used also by DTLS and POPS/IMAPS/SMTPS/FTPS */
```

Comment:
Since even QUIC uses a generic TLS handshake, why not 
name it only `tls` ?

Reply:
I simply copied the name already used for `flow->protos.tls_quic`.
I could change both of them. What do you think?

Reply:
Not necessary. I guess the change-set might be huge.